.changeset/smart-ads-rush.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/core-components': patch
+---
+
+Make the documentation pages printable (also handy for exporting to pdf when go-green)
```

Comment:
you can use PDF in uppercase to silence this error i think

Reply:
I am not sure about the go-green wording :D `printable (including as PDF)` is clearer to me personally at least!